src/main/java/model/FlowerBunch.java

```diff
@@ -0,0 +1,10 @@
+package model;
+
+public class FlowerBunch extends Flower {
```

Comment:
Bunch is NOT a Flower
Bunch is a composition (not inheritance), or is an attribute of a Bouquet
if you mean Bunch is a special flower with many small flowers then it could be ok